kernel/sched.c

```diff
@@ -502,7 +502,7 @@ void z_thread_priority_set(struct k_thread *thread, int prio)
 	}
 }
 
-static inline int resched(void)
+static inline int resched(u32_t key)
```

Comment:
const u32_t key ?

Reply:
this is not a pointer, does it matter really?

Reply:
This is function which tests some conditions. Since you testing variable, const is IMHO what we should have here. But as you said: this is not a pointer, so my comment is just suggestion.